runtimes/pythonrt/ak_runner/deterministic.py

```diff
@@ -122,4 +146,4 @@ def is_determinstic(fn):
     datetime.time.utcoffset,
     datetime.time.dst,
     datetime.time.tzname,
-}
+} | builtin_types
```

Comment:
Nit: I think it's cleaner and easier to understand that `modules` includes `builtin_types` when you reverse the order, i.e.:

````
modules = builtin_types | {
   ...
}
```

But that's optional.

Reply:
Good idea. Done.